src/backend/marsha/core/templates/core/form_autosubmit.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
I think it should have lti_select in its filename.

Reply:
The idea was to keep it generic : this template submits the form, this is not related to LTI.
But if you want I'll change its name.

Reply:
Well it does contain "lti_selected" in its body :wink: 
> document.getElementById("lti_select").submit();

Reply:
Good catch, I'll remove it :)

Reply:
:+1: